### ISSUE TITLE
The main web page does not work and the tutorial cannot be followed

### ISSUE THREAD

**Issue by FTamas77** [JRCSTU/CO2MPAS-TA#52]
I found this tool a few days ago after a longer search and this is the tool I need. I want to generate test data for my research. I have only one requirement: there should be some intervention at a given time. I need to know this time point. It can be, for example, a change in the mass. How can I gain simulation results before and after this intervention?

Most of the links in the tutorial are broken and hard to follow for me (I'm really not an expert of the field). My biggest problem the main web page cannot be loaded. I always redirected here:

http://127.0.0.1:5000/?url=http://127.0.0.1:5000/co2mpas_conventional.html

![image](https://github.com/JRCSTU/CO2MPAS-TA/assets/76584703/cf69220e-4fca-4bd5-900d-e541090ca6f1)

How can I create a single table where the input and the output time-series data are together?

**Comment by FTamas77** [comment]
I found some errors, maybe this one can related the main page:

2023-11-09 11:49:00,327:ERROR:schedula.utils.sol:Failed DISPATCHING 'CO2MPAS model/calibrate_with_wltp_h/clutch_torque_converter_model/clutch/calibrate_clutch_speed_model' due to:
  RuntimeError("scikit-learn estimators should always specify their parameters in the signature of their __init__ (no varargs). <class 'co2mpas.core.model.physical.engine._thermal._SelectFromModel'> with constructor (self, *args, in_mask=(), out_mask=(), **kwargs) doesn't  follow this convention.")
Traceback (most recent call last):

**Comment by FTamas77** [comment]
What I've done so far: tried to install different Python versions to solve the problem without no success. Could you export your list of used Python packages? If I had the same environment, then I should have the same running app. And probably, I could use the demo file (with examples). So, my problem would have been solved.

**Comment by vinci1it2000** [comment]
We have released a new version of `co2mpas` that should solve your issues.  

You can install it using the command `pip install co2mpas -U` from co2mpas command line.

If the problem persist reopen the issues and describe again the issue with the full log file.

**Comment by FTamas77** [comment]
Hi! Thanks.

When I want to open the output, I have this:
![image](https://github.com/JRCSTU/CO2MPAS-TA/assets/76584703/67b81135-42da-4de6-bf28-b4b06363d70f)


And the interesting log:

Co2 params selector warning: Models (calibration_status,co2_params_calibrated,engine_idle_fuel_consumption,initial_friction_params) to predict nedc_h,wltp_h failed the calibration.
2023-11-18 14:15:26,344: INFO:co2mpas.core.write.convert:Failed collecting installation info.
[WinError 2] The system cannot find the file specified

Everything else looks good. I mean I can open the output and use it.

**Comment by FTamas77** [comment]
@vinci1it2000 But I don't need to re-open this issue. Thanks again.
